react-components/src/components/Checkout/__test__/useOverview.test.js

```diff
@@ -257,4 +280,153 @@ describe('useOverview', () => {
+                            carrier_code: 'flatrate',
+                            carrier_title: 'Flat Rate',
+                            method_code: 'flatrate',
+                            method_title: 'Fixed'
+                        }
+                    }
+                ],
+                available_payment_methods: [
+                    {
+                        code: 'checkmo',
+                        title: 'Check / Money order'
+                    }
+                ],
+                selected_payment_method: {
+                    code: 'checkmo',
+                    title: 'Check / Money order'
+                },
+                billing_address: {
+                    city: 'Bucuresti',
+                    country: {
+                        code: 'RO'
+                    },
+                    lastname: 'Some',
+                    firstname: 'User',
+                    region: {
+                        code: 'B'
+                    },
+                    street: ['Some address'],
+                    postcode: '012321',
+                    telephone: '000000000'
+                },
+                applied_coupon: null,
+                total_quantity: 1,
+                items: [
+                    {
+                        __typename: 'SimpleCartItem',
+                        quantity: 1,
+                        prices: {
+                            price: {
+                                currency: 'USD',
+                                value: 22
+                            },
+                            row_total: {
+                                currency: 'USD',
+                                value: 22
+                            }
+                        },
+                        product: {
+                            name: 'Radiant Tee-M-Orange',
+                            sku: 'WS12-M-Orange',
+                            thumbnail: {
+                                url:
+                                    'http://localhost:8080/media/catalog/product/cache/3431b869eac756ac84605eb6066d9c1e/w/s/ws12-orange_main_2.jpg'
+                            }
+                        }
+                    }
+                ]
+            },
+            errorMessage: null,
+            couponError: null
+        };
+
+        const mockCheckoutState = {
+            flowState: 'form',
+            editing: null,
+            isEditingNewAddress: false,
+            billingAddressSameAsShippingAddress: false
+        };
+
+        const { getByRole } = render(
+            <CartProvider initialState={mockCartState}>
+                <CheckoutProvider initialState={mockCheckoutState}>
+                    <Wrapper />
+                </CheckoutProvider>
+            </CartProvider>
+        );
+
+        expect(getByRole('button')).not.toBeUndefined();
+        await act(async () => fireEvent.click(getByRole('button')));
+
+        expect(mse.context.setOrder).toHaveBeenCalledTimes(1);
```

Comment:
IMO it makes sense to check for the value the `setOrder` method was called with here.